.github/workflows/ci.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         id: postgres
 
       - name: Test PostGIS with connection URI
-        run: psql -v ON_ERROR_STOP=1 -c 'SELECT PostGIS_Full_Version();' '${{ steps.postgres.outputs.connection-uri }}'
+        run: psql -v ON_ERROR_STOP=1 -c 'SELECT PostGIS_Full_Version();' ${{ steps.postgres.outputs.connection-uri }}
```

Comment:
why remove the quotes?  I think the `${{...}}` expansion is done by github, not by bash

Reply:
Oops. You are right.